### PR TITLE
[Improve][all] change Log to @Slf4j

### PR DESCRIPTION
**Pull request by liugddx**
## Purpose of this pull request

This PR contains all the log type substitutions, use @SLF4J unified mode.There are no other logical changes here,but it's a big change,so ask the community to review in detail.

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

**Review by CalvinKirs — COMMENTED** [review]
LGTM, but there are some unrelated modifications, are you able to restore it? Also, I didn't check if it has been completely replaced

**Comment by liugddx** [comment]
> LGTM, but there are some unrelated modifications, are you able to restore it? Also, I didn't check if it has been completely replaced

Done.

**Review by EricJoy2048 — DISMISSED** [review]
LGTM

File: seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-druid/src/main/java/org/apache/seatunnel/flink/druid/sink/DruidOutputFormat.java [hunk 1]
```diff
@@ -179,9 +178,9 @@ private ParallelIndexIOConfig parallelIndexIOConfig() {
                 null,
                 new InlineInputSource(this.data.toString()),
                 new CsvInputFormat(
-                        Arrays.asList("name", timestampColumn),
+                    Arrays.asList("name", timestampColumn),
                         "|",
-                        null,
+                           null,
```
Comment (hailin0) [inline seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-druid/src/main/java/org/apache/seatunnel/flink/druid/sink/DruidOutputFormat.java]: revert？

**Review by hailin0 — COMMENTED** [review]
LGTM

File: seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-druid/src/main/java/org/apache/seatunnel/flink/druid/sink/DruidOutputFormat.java (hunk 1 above)
Comment (liugddx) [reply seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-druid/src/main/java/org/apache/seatunnel/flink/druid/sink/DruidOutputFormat.java]: Done.Thank you for your review.

**Review by hailin0 — DISMISSED** [review]

**Comment by EricJoy2048** [comment]
Please resolve conflicts.

**Comment by liugddx** [comment]
> Please resolve conflicts.

Done.

**Comment by EricJoy2048** [comment]
Please waiting the CI complete. I will merge it after CI finished.

**Comment by liugddx** [comment]
please rerun the CI. @EricJoy2048

**Comment by CalvinKirs** [comment]
Regarding Connctor V1, do we still need to make replacements?For me, it doesn't make much sense.

**Review by CalvinKirs — APPROVED** [review]

**Review by EricJoy2048 — APPROVED** [review]
+1

**Pull request merged**